invokeai/frontend/web/public/locales/en.json

```diff
@@ -1684,6 +1680,8 @@
         "postProcessingMissingModelWarning": "Visit the <LinkComponent>Model Manager</LinkComponent> to install a post-processing (image to image) model.",
         "missingModelsWarning": "Visit the <LinkComponent>Model Manager</LinkComponent> to install the required models:",
         "mainModelDesc": "Main model (SD1.5 or SDXL architecture)",
+        "outputTooLargeShort": "Output is too large to upscale",
```

Comment:
The input is too large to upscale, not the output... Not sure how to word this... maybe `Upscale output too large` or `Upscale settings exceed max size limit`?